core_perception/range_vision_fusion/launch/range_vision_fusion.launch

```diff
@@ -1,23 +1,24 @@
 <launch>
   <arg name="detected_objects_range" default="/detection/lidar_detector/objects"/>
   <arg name="detected_objects_vision" default="/detection/image_detector/objects"/>
-  <arg name="camera_info_src" default="/camera_info"/>
   <arg name="min_car_dimensions" default="[3,2,2]"/>
   <arg name="min_person_dimensions" default="[1,2,1]"/>
   <arg name="min_truck_dimensions" default="[4,2,2]"/>
   <arg name="sync_topics" default="false"/>
   <arg name="overlap_threshold" default="0.6"/>
   <arg name="use_vector_map" default="false"/>
   <arg name="namespace" default="/detection/fusion_tools"/>
+  <arg name="camera_info_src" default="/camera_info"/>
 
   <node name="range_vision_fusion_01" pkg="range_vision_fusion" type="range_vision_fusion" output="screen">
     <param name="detected_objects_range" value="$(arg detected_objects_range)"/>
     <param name="detected_objects_vision" value="$(arg detected_objects_vision)"/>
-    <param name="camera_info_src" value="$(arg camera_info_src)"/>
     <param name="min_car_dimensions" value="$(arg min_car_dimensions)"/>
     <param name="min_person_dimensions" value="$(arg min_person_dimensions)"/>
     <param name="sync_topics" value="$(arg sync_topics)"/>
     <param name="overlap_threshold" value="$(arg overlap_threshold)"/>
+    <remap from="$(env CARMA_ENV_NS)/camera_info" to="$(arg camera_info_src)" /> 
```

Comment:
you shouldn't need the $(env CARMA_ENV_NS) here as this is a local remapping. Also this would throw an error if that environment variable did not exist.

Reply:
Why did you remove the camera_info_src argument?